Makefile

```diff
@@ -115,13 +141,13 @@ deploy:
         echo; \
         echo -n \* Waiting for KubeDirector to start...; \
         sleep 3; \
-        retries=20; \
-        while [ $$retries ]; do \
```

Comment:
This change is fine, but curious about motivation. Not working correctly on your system?

Reply:
expr manpage:

       Exit status is 0 if EXPRESSION is neither null nor 0, 1 if EXPRESSION is null or 0, 2 if EXPRESSION is syntactically invalid, and 3 if an error occurred.

When this expression went to zero, " retries=`expr $$retries - 1`", the exit status of the expr command was 1 and the make command exited because it thought there was a command failure.